app/src/main/java/tech/relaycorp/courier/data/database/StorageConverter.kt

```diff
@@ -0,0 +1,12 @@
+package tech.relaycorp.courier.data.database
+
+import androidx.room.TypeConverter
+import tech.relaycorp.courier.data.model.StorageSize
+
+class StorageConverter {
```

Comment:
Minor suggestion: What do you think if we call it `StorageSizeConverter`? `StorageConverter` suggests a different thing to me.
```suggestion
class StorageSizeConverter {
```

Reply:
Yup, changed.